doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1117,6 +1122,11 @@ Other libraries
 
   * Added support for the nRF54L15 SoC.
 
+* :ref:`dult_readme` library:
+
+  * Added DULT motion detector feature integration.
```

Comment:
Nit: perhaps we should avoid the word integration as integration can occur at the higher level of the stack (at the FMDN extension level / at the sample level)